masschallenge/django-accelerator
# AC-7594: Add user role: Deferred Mentor

**Changes made to [AC-7594](https://masschallenge.atlassian.net/browse/AC-7594):**
- Added Deferred Mentor user role

**How to test:**
- Check out to this branch and sibling PR.
- Run `make migrate` to apply the migration
- Check [user role admin page](http://localhost:8181/admin/mc/userrole/) and notice that deferred mentor is one of the user roles.
- Visit [program role admin page](http://localhost:8181/admin/mc/programrole/?q=deferred+mentor) and view the past programs backfilled with the new user role

Sibling PR found on [accelerate](https://github.com/masschallenge/accelerate/pull/2592)
